ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -352,6 +352,23 @@ def frac(x, name=None):
     return ivy.subtract(x, y)
 
 
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def logcumsumexp(x, axis=None, dtype=None, name=None):
+    if len(x.shape) == 0:
+        ret = x
+    else:
+        # x is input
+        # name is out
+        original_dtype = dtype
+        exp_input = ivy.exp(x.astype("float64"))
+        summed_exp_input = ivy.cumsum(exp_input, axis=axis)
+        ret = ivy.log(summed_exp_input).astype(original_dtype)
+    if ivy.exists(name):
+        ivy.inplace_update(name, ret)
+    return ret
```

Comment:
doesn't return expected values - 
```
from ivy.functional.frontends.paddle import logcumsumexp
import ivy
ivy.set_backend("paddle")

x = ivy.array([[-1.5, 0., 2.], [3., 1.2, -2.4]])
print(logcumsumexp(x))

import paddle

x = paddle.to_tensor([[-1.5, 0., 2.], [3., 1.2, -2.4]])
print(paddle.logsumexp(x))
```

Reply:
I don't know if you realised but your showing an implementation of ```logsumexp``` on paddle but the ```logcumsumexp```  on the ivy version . I also added ```float16``` to supported data types. I tested it using the values on the paddle api docs and it returned with the correct values though with more decimal places :
<img width="658" alt="Screenshot 2023-07-27 at 19 54 59" src="https://github.com/unifyai/ivy/assets/96082317/44bfa833-8d04-4cce-91f1-f35560115cb1">
<img width="951" alt="Screenshot 2023-07-27 at 19 55 20" src="https://github.com/unifyai/ivy/assets/96082317/186f60e0-a63e-4f2c-ae39-2ca17fdb3dd8">